### DISCUSSION TITLE
how i get an ip for my server

### DISCUSSION THREAD

**Discussion by justoramm** [itzg/docker-minecraft-server#2671] [Q&A]
i dont know how to get an id for mi friends to connect to my server i tried installing playit.gg in examples/paper-build-plugins/plugins
and starting the server with the command "docker run -d -v /path/on/host:/data \-e TYPE=PAPER \-p 25565:25565 -e EULA=TRUE --name mc itzg/minecraft-server" but it doesn't work

**Comment by itzg** [comment]
I'm not familiar with playit, so you'll need to contact them for support. 

Otherwise, the remaining steps are not specific to docker, so search for 

https://www.google.com/search?q=publish+minecraft+server+home+network

where the LAN IP address will be the host where docker is running.
